keep-ui/app/(keep)/settings/settings.client.tsx

```diff
@@ -73,24 +73,24 @@ export default function SettingsPage() {
       newSelectedTab === "users"
         ? 0
         : newSelectedTab === "webhook"
-        ? 1
-        : newSelectedTab === "smtp"
-        ? 2
-        : 0;
+          ? 1
+          : newSelectedTab === "smtp"
+            ? 2
+            : 0;
     const userSubTabIndex =
       newUserSubTab === "users"
         ? 0
         : newUserSubTab === "groups"
-        ? 1
-        : newUserSubTab === "roles"
-        ? 2
-        : newUserSubTab === "permissions"
-        ? 3
-        : newUserSubTab === "api-keys"
-        ? 4
-        : newUserSubTab === "sso"
-        ? 5
-        : 0;
+          ? 1
```

Comment:
😵‍💫

Reply:
Feel your pain, but this is just a prettier again =)